### ISSUE TITLE
Overflow scroll

### ISSUE THREAD

**Issue by visualjerk** [v1Labs/protovue#40]
Nice work! The only thing bugging me is that every cell has set overflow: scroll

This leads to a pretty ugly experience. You should consider getting rid of this property.

**Comment by DaneLyons** [comment]
I agree Jörg. I've been using Protovue and running into the same issues. I think I'll remove the overflow default and make it configurable. Thanks for the input!
